config.go

```diff
@@ -29,8 +29,12 @@ var (
 	defaultVSPFee         = 3.0
 	defaultNetwork        = "testnet"
 	defaultHomeDir        = dcrutil.AppDataDir("vspd", false)
+	defaultLogDirName     = "logs"
+	defaultLogDir         = filepath.Join(defaultHomeDir, defaultLogDirName)
 	defaultConfigFilename = "vspd.conf"
 	defaultConfigFile     = filepath.Join(defaultHomeDir, defaultConfigFilename)
+	defaultDataName       = "data"
```

Comment:
```suggestion
	defaultDataDirName       = "data"
```

It's not the name of some data, but the name of a directory.